skimage/future/tests/test_graph.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+def test_future_graph_import_error():
+    error_msg = "The API in skimage.future.graph was moved into skimage.graph"
+    with pytest.raises(ModuleNotFoundError, match=error_msg):
+        import skimage.future.graph
+
+    with pytest.raises(ModuleNotFoundError, match=error_msg):
+        from skimage.future import graph
+
+    with pytest.raises(ModuleNotFoundError, match=error_msg):
+        from skimage.future.graph import rag
```

Comment:
No way the above tests pass and this one fails.

Reply:
But they do. See https://github.com/scikit-image/scikit-image/pull/6674#discussion_r1073370525. :sweat_smile: